src/howitz/endpoints.py

```diff
@@ -1,32 +1,64 @@
-from enum import StrEnum
+import os
 
-from flask import Flask, render_template, request
+import flask
+from flask import Flask, render_template, request, make_response
+from flask_login import LoginManager, login_required, login_user, current_user
 from flask_assets import Bundle, Environment
+from logging.config import dictConfig
 
-import logging
+from enum import StrEnum
```

Comment:
I know you've locked howitz to python 3.11 or newer, but zinolib has a StrEnum in zinolib.compat that works on older pythons.

Reply:
If all you need is StrEnum you could as I said use the one in zinolib.compat, or just copy it out of there.

Reply:
(I wanted an excuse to use the new match-statement when converting curitz.cli.poll but figured out it would be better with a class. Ergo, zinolib still works on older pythons.)